curvefs/src/mds/fs_storage.cpp

```diff
@@ -369,6 +396,21 @@ bool PersisKVStorage::LoadAllFs() {
         }
 
         fs_.emplace(fsInfo.fsname(), std::move(fsInfo));
+
+        // load fs usage to cache
+        FsUsage fsUsage;
+        auto ret = GetFsUsage(fsInfo.fsname(), &fsUsage, false);
+        if (ret == FSStatusCode::NOT_FOUND) {
+            SetFsUsage(fsInfo.fsname(), fsUsage);
```

Comment:
Is there any tool that can calculate the usage of existing file systems?

Reply:
planned